pool_configs/aion.json

```diff
@@ -2,10 +2,11 @@
     "enabled": true,
     "testnet": true,
     "coin": "aion.json",
-    "logLevel": "debug",
-    "__address_aion":    "0x604076f02ab252f3927e6a8721e2a18526e08182fd7b476a8f054765a60803b6",
+
+    "__address_aion":    "6a156bA44e87E1d621C266687eA6B63B2283cA5c",
```

Comment:
what is this address used for? should we have placeholders instead?

Reply:
Yup, you're right. I'll change it